### PR TITLE
[shelly] Add support for Shelly Wall Dimmer US  and Wall Display

### LINKED ISSUE #15052: [shelly] channel totalKWH and totalReturned show wrong values
Fixed channel totalKWH and totalRetruned: correct value is kw/h (API value needs to be converted from Watt/Min to kw/h)

### LINKED ISSUE #15147: [shelly] Support for Shelly Dimmer US
Add support for Shelly Dimmer US

### LINKED ISSUE #15148: [shelly] Support for Shelly Wall Display
Support for Shelly Wall Display added

### LINKED ISSUE #15149: [shelly] Support for Shelly BLU Button 1 and BLU Door/Window
Support for Shelly BLU series of devices (Button 1 and Door/Window)
Those devices are integrated using a regular Shelly Plus or Pro device as a bluetooth hub. The binding automatically installs a script on the device to forward Bluetooth events to the OH binding. The features needs to be enabled in the thing config.

### LINKED ISSUE #15150: [shelly] New channel to reset Energy Meter Data for Shelly Pro 3EM
The Pro 3 EM offers the option to clear captured energy data. A new Switch channel resetTotals allows to clear that information. Sending ON to the channel clears the data.

### LINKED ISSUE #15151: [shelly] Internal temp channel created for RGBW2, Bulb and Vintage even the devices doesn't provide the internal temp
The binding creates the internalTemp channel even the device doesn't provide that information

### LINKED ISSUE #15152: [shelly] Incorrect UOM types for energy related channel in relays
openHAB 4.0 introduces a stricter validation of channel updates. Some of the relay channels need to be fixed, because item type and updates of Quantity Types by the binding didn't matched and therefore channel was never updated

### LINKED ISSUE #15204: [shelly] Don't create accumulatedXXX channels with only 1 meter
It doesn't make sense to create the accumulatedXXX channels if the device only has 1 meter

### LINKED ISSUE #15206: [shelly] No sensor vibration for Shelly Motion2
The channel sensors#vibration is not created for Motion 2 device

### PR DESCRIPTION
**Pull request by markus7017**
This PR adds support for Shelly Wall Dimmer US and the new Wall Display 4"

### CODE REVIEW

**Comment by markus7017** [comment]
@kaikreuzer do you habe time to do the review?

**Comment by kaikreuzer** [comment]
@markus7017 Do you consider this PR ready? I'm asking because the title still says [WIP].

**Comment by markus7017** [comment]
@kaikreuzer yws, I already removed the label, bit forgot the tag on the title
we are ready to go

File: bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_relay.xml [hunk 1]
```diff
@@ -533,7 +549,11 @@
 		<item-type>Number:Energy</item-type>
 		<label>@text/channel-type.shelly.meterTotal.label</label>
 		<description>@text/channel-type.shelly.meterTotal.description</description>
-		<category>Energy</category>
+		<!-- category>Energy</category>
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_relay.xml:552]: Did you put this into comments on purpose?

**Review by kaikreuzer — APPROVED** [review]
Thanks, lgtm!
Just one last question before I will merge, see below.

File: bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_relay.xml (hunk 1 above)
Comment (markus7017) [reply bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_relay.xml:552]: fixed

**Comment by markus7017** [comment]
@kaikreuzer I applied the changes, but screwed up the PR with wrong git operations
could you help or should I close this PR and recreate a new one

**Comment by markus7017** [comment]
@kaikreuzer I rebased with main and that seem to fix the problem

**Review by kaikreuzer — APPROVED** [review]
Thanks!

**Comment by markus7017** [comment]
thanks @kaikreuzer

**Pull request merged**